client/my-sites/plans-features-main/hooks/experiments/use-longer-plan-term-default-experiment.ts

```diff
@@ -0,0 +1,47 @@
+import { useExperiment } from 'calypso/lib/explat';
+
+const useTermFromExperimentVariant = (
+	experimentVariant: string | null | undefined
+): string | null => {
+	switch ( experimentVariant ) {
+		case 'default_to_three_year_plans':
+			return '3yearly';
+		case 'default_to_two_year_plans':
+			return '2yearly';
+		default:
+			return null;
+	}
+};
+
+/**
+ * Returns:
+ * - the term as yearly, 2yearly, 3yearly (etc.) to be used as default in the plans
+ * page based on the experiment variant
+ * - OR null/undefined if control variant or if the experiment is still loading (Subject to change)
+ *
+ * This hook although used for the experiment, it can be refactored in the end to
+ * define the default term in the grid/plans page.
+ *
+ */
+const useLongerPlanTermDefaultExperiment = (): {
+	isLoadingExperiment: boolean;
+	term?: string | null;
+	// TODO: Consider removing this and always return concrete term values (where undefined/null would mean no term savings)
```

Comment:
I'll study this a bit more. I want to make the "clean up" phase after the experiment as easy as possible, e.g., this hook remaining to define the default term/interval type.